Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -45,6 +45,13 @@
                          *globalValidation
                          *hltvalidation)
 
+# fastsim: remove bits that depend on digis, tracker simhits or tracker rechits
+def _validation_fastSimMods(sequence):
+    sequence.remove(globaldigisanalyze)
+    sequence.remove(globalhitsanalyze)
+    sequence.remove(globalrechitsanalyze)
+eras.fastSim.toModify(validation,func=_validation_fastSimMods)
```

Comment:
I'd really like to deprecate the `toModify` which takes a functional form since it would be easy to abuse. I think the `toReplaceWith` makes for a clearer intent and would allow me more opportunities for changes to the `Modifier` system in the future.

That being said, I personally would prefer that instead of constantly adding new features we attempt to leverage the existing abilities of python. e.g.
```python
eras.fastSim.toModify(someSequence, lambda x: map(x.remove, [b,d] ) )
```
or, equivalently
```python
eras.fastSim.toModify(someSequence, lambda x: [ x.remove(y) for y in [b,d] ] )
```